src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -35,6 +36,14 @@ const FacilitySearch = props => {
     return facilities?.length < pagination.totalEntries;
   };
 
+  const ariaLiveMessage = () => {
+    if (newFacilitiesCount > 0) {
+      const facilitySuffix = newFacilitiesCount === 1 ? 'y' : 'ies';
+      return `${newFacilitiesCount} new facilit${facilitySuffix} loaded`;
```

Comment:
Lets get rid of the magic string here. You can reference the `replaceStrValues` I have in the app to replace those dynamic values.